sender/http.go

```diff
@@ -166,7 +167,16 @@ func (ht *HTTP) Send(c *skogul.Container) error {
 		}
 		ht.client = &http.Client{Transport: &tran, Timeout: ht.Timeout.Duration}
 	})
-	resp, err := ht.client.Post(ht.URL, "application/json", &buffer)
+	req, err := http.NewRequest("POST", ht.URL, &buffer)
```

Comment:
I get the transition to NewRequest, but for the HTTP sender itself we need to provide the Content-Type application/json at least. It can be done in init, though: After headers are parsed, check if Content-Type is set and add application/json if it isn't set. That will give the user the ability to override the content-type too.